system/Config/Services.php

```diff
@@ -185,6 +187,33 @@ public static function curlrequest(array $options = [], ResponseInterface $respo
 
 	//--------------------------------------------------------------------
 
+	/**
+	 * The Encryption class provides two-way encryption.
+	 *
+	 * @param mixed   $config
+	 * @param boolean $getShared
+	 *
+	 * @return EncrypterInterface Encryption handler
+	 */
+	public static function encrypter($config = null, $getShared = false)
+	{
+		if ($getShared === true)
+		{
+			return static::getSharedInstance('encrypter', $config);
+		}
+
+		if ($config !== null && is_object($config))
+		{
+			$config = (array) $config;
+		}
+
```

Comment:
We should probably load the default config here if none is provided. That would match how the rest of them handle it.

Reply:
ok